src/xenia/gpu/spirv_shader_translator.cc

```diff
@@ -292,6 +298,24 @@ void SpirvShaderTranslator::StartTranslation() {
       b.createStore(vec4_float_zero_, ptr);
     }
 
+    point_size_ = b.createVariable(spv::StorageClass::StorageClassOutput,
+                                   float_type_, "point_size");
+    b.addDecoration(point_size_, spv::Decoration::DecorationLocation, 17);
+    // Set default point-size value (-1.0f, indicating to the geometry shader
+    // that the register value should be used instead of the per-vertex value)
+    b.createStore(point_size_, b.makeFloatConstant(-1.0f));
+
+    point_coord_ = b.createVariable(spv::StorageClass::StorageClassOutput,
```

Comment:
The fragment shader always generates code that reads point_coord (the ps_param_gen code path). We might be able to get rid of it if we knew what the primitive type was at shader translation time, since afaict it only applies to point lists, but the shader translator doesn't seem to have that information available.

In any case, it's probably cheap enough to just load a extra zero into an output to not really worry about.